scripts/error-codes/codes.json

```diff
@@ -271,7 +269,6 @@
   "269": "Profiler must specify an \"id\" string and \"onRender\" function as props",
   "270": "The current renderer does not support persistence. This error is likely caused by a bug in React. Please file an issue.",
   "271": "Failed to replay rendering after an error. This is likely caused by a bug in React. Please file an issue with a reproducing case to help us find it.",
-  "272": "The current renderer does not support hydration. This error is likely caused by a bug in React. Please file an issue.",
```

Comment:
Will deleting these duplicate error messages cause older versions of React to throw error codes that can't be looked up in our error decoder? Seems like it would?

Reply:
I just ran `extract-errors` and this was autogenerated. Should I re-add the old codes in?

Reply:
Hm 🤔  I don't know!

Reply:
I think it depends on if we ever made releases of React that logged the error code numbers you've deleted.

The [website error decoder](https://github.com/reactjs/reactjs.org/blob/master/src/components/ErrorDecoder/ErrorDecoder.js) uses a [custom Gatsby plug-in](gatsby-source-react-error-codes) we wrote which embeds [whatever the latest error codes file is](https://github.com/reactjs/reactjs.org/blob/master/plugins/gatsby-source-react-error-codes/gatsby-node.js#L3-L4) at the time we are updating the website (which we do with each release).

So I think this means that if an older version of React logged e.g. error code 272 rather than 305, then the website would not have any idea how to decode that error.

Reply:
I can re-add those entries and then we can have a discussion in our next sync about this.